CustomScripts/focus-between-floats.sh

```
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Store the list of floating window IDs into an array and sorted descending
-floating_windows=($(yabai -m query --windows --space | jq '.[] | select(.["is-floating"] == true) | .id' | sort -r))
+floating_windows=($(yabai -m query --windows --space | jq '.[] | select((.["is-floating"] == true) and (.["is-visible"] == true)) | .id' | sort -r))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)